compiler/src/language_server/inlayhint.re

```diff
@@ -0,0 +1,89 @@
+open Grain;
+open Compile;
+open Grain_parsing;
+open Grain_utils;
+open Grain_typed;
+open Grain_diagnostics;
+open Sourcetree;
+open Lsp_types;
+
+// https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#inlayHintParams
+
```

Comment:
```suggestion
// https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#inlayHintParams
```

Reply:
I've stared at this for ages, what's the code change?

Reply:
It's just removing the space so the comment is attached to a node.

Reply:
Ah, done.